libethereum/BlockChain.cpp

```diff
@@ -364,21 +364,21 @@ pair<ImportResult, ImportRoute> BlockChain::attemptImport(bytes const& _block, O
 	}
 	catch (UnknownParent&)
 	{
-		return make_pair(ImportResult::UnknownParent, make_pair(h256s(), h256s()));
+		return make_pair(ImportResult::UnknownParent, ImportRoute());
```

Comment:
Indeed it's not the same. When creating temporaries, unless the C++ actually prefers one over the other (i.e. it's considering deprecating one or compiler support for the other is patchy), I consider both to be acceptable.